docs/rule/simple-modifiers.md

```diff
@@ -0,0 +1,34 @@
+# simple-modifiers
+
+This rule strongly advises against using the `{{modifier}}` helper with its first argument being either:
+
+- a string literal (.e.g. `{{modifier "tracking-interaction"}}`)
+- a variable in the template's JS backing class context (e.g. `{{modifier this.trackingInteraction}}`)
```

Comment:
Shouldn't there be two invalid test cases, one for each of these?

Reply:
These are the valid use cases - agree that the wording doesn't seem quite right here.

Reply:
I've updated the wording with your suggestion, @lin-ll.

@bmish these are correct cases. I've updated the wording above to reflect that.